main.py

```diff
@@ -164,10 +165,20 @@ def is_neutral(status):
     query_result = data["query_result"]
     data = query_result["data"]
     rows = data["rows"]
+
+    # Determine the date range of this dataset.
+    min_timestamp = min([row.pop("min_timestamp") for row in rows])
```

Comment:
```suggestion
    min_timestamp = min(row["min_timestamp"] for row in rows)
```

Reply:
I saw this below, that's why I used `.pop()`
https://github.com/mozilla/remote-settings-uptake-health/blob/b78ba74710bff6f56e55bea9989094b52a439912/main.py#L210-L215

Reply:
I don't get it. The `v` is unchanged and wouldn't be affected if there's one or two more keys. Or am I still missing something?

Reply:
won't `min_timestamp` and `max_timestamp` be considered bad status?

Reply:
Oh my! You're right. I thought the statuses was under a differrent sub-key.

Reply:
Ignore my suggestion but I think it might be good to add a code comment that rerminds us *why* we're using `.pop` since even I managed to get confused by it. Uh. /me tired.